### PR TITLE
Cleanup info log messages on startup

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

File: cilium-health/launch/endpoint.go [hunk 1]
```diff
@@ -52,16 +52,6 @@ func logFromCommand(cmd *exec.Cmd, netns string) error {
 			scopedLog.Debugf("%s", in.Text())
 		}
 	}()
-	stderr, err := cmd.StderrPipe()
```
Comment (joestringer) [inline cilium-health/launch/endpoint.go]: I think the right answer here is probably to restrict the permissions set to only one of the daemons.

In the log you can see that one of the logs states "netns=cilium-health", while the other doesn't. Both the cilium-health instance that runs on the host, and the cilium-health that runs in a namespace are setting the permissions of this path, but they don't both need to - only one of them is actually hosting that socket. The permissions set should be contingent upon serving the unix api.

**Comment by tgraf** [comment]
Ginkgo failed because of what appears to be a k8s interaction flake:
```
16:28:18     Error: Cannot get security id from k8s pod name: unable to get pod app3-596cc57c5b-6kxgd in namespace default
16:28:18     command terminated with exit code 1
16:28:18     level=info msg="Waiting for k8s api-se
```

**Comment by tgraf** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Comment by tgraf** [comment]
Discussed with @joestringer, dropping the duplicate log removal commit.

File: cilium-health/launch/endpoint.go (hunk 1 above)
Comment (tgraf) [reply cilium-health/launch/endpoint.go]: fixed by dropping this commit

**Comment by tgraf** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Pull request merged**